### ISSUE TITLE
enhancement: tree -> [name].txt

### ISSUE THREAD

**Issue by Jamesbarford** [kepler471/listty#6]
Feature to convert the tree into a .txt representation

**Comment by Jamesbarford** [comment]
### Thoughts
- Do we want to include the root node? You can't edit it and it is just to kick off the tree.
- Should we split the root node into a separate class which houses the `*[]item`?
  - If we did that we could then have the traversal methods on the Root nodes class

As this output while "correct" doesn't make much sense.

<img src="https://user-images.githubusercontent.com/29375815/93707209-616ab900-fb24-11ea-83c0-5d089a0cbb23.png" width="400" />

**Comment by Jamesbarford** [comment]
I think this makes more sense:

<img src="https://user-images.githubusercontent.com/29375815/93708786-02f80780-fb31-11ea-9e95-f75d9fd60ac8.png" width="400" />

**Comment by kepler471** [comment]
We did speak about creating a new type for root node, to help with limitations on how it could be used. The only issue is that our items point to items as their parents, so we would have to make a special case for the top level items. Can we make better use of the 'Home' field in item? It was put in to help determine root/home level.

I agree that it should not be printed with the items (as in that last image you posted). Down the line, in a future enhancement, we can use some tcell features (views) to display our level at the top, so 'Root' or if we entered the '£' note, it would say '£'. Very much like workflowy, and to the hacky ways we currently have of displaying where we are in our tree.
